docs/user/build-customization.rst

```diff
@@ -631,6 +631,35 @@ Here is an example configuration file:
          html:
            - asciidoctor -D $READTHEDOCS_OUTPUT/html index.asciidoc
 
+Using pydoctor
+~~~~~~~~~~~~~~
+
+`Pydoctor <https://github.com/twisted/pydoctor>`_ is an easy-to-use standalone API documentation tool for Python.
+Here is an example configuration file:
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+   version: 2
+   build:
+     os: "ubuntu-22.04"
+     jobs:
+       install:
+         - pip install pydoctor
+       build:
+         html:
+           - |
+             pydoctor \
+               --project-name='my project' \
+               --project-version=${READTHEDOCS_GIT_IDENTIFIER} \
+               --project-url=${READTHEDOCS_GIT_CLONE_URL%*.git} \
+               --html-viewsource-base=${READTHEDOCS_GIT_CLONE_URL%*.git}/tree/${READTHEDOCS_GIT_COMMIT_HASH} \
+               --html-base-url=${READTHEDOCS_CANONICAL_URL} \
+               --html-output $READTHEDOCS_OUTPUT/html/ \
+               --docformat=restructuredtext \
+               --intersphinx=https://docs.python.org/3/objects.inv \
+               ./src/my_project
```

Comment:
This feels a bit specific and not generic -- is there not a config file this would normally live in? I'm happy to have a pydoctor example, but would be good to make it a bit more minimal, and link to an external resource (pydoctor docs or blog post) with the additional config options.

Reply:
Yea, I think that makes sense. :+1: